test_runner/fixtures/pageserver/http.py

```diff
@@ -60,6 +60,7 @@ class HistoricLayerInfo:
     lsn_start: str
     lsn_end: Optional[str]
     remote: bool
+    l0: Optional[str]
```

Comment:
```suggestion
    l0: Optional[bool]
```

or maybe even `l0: bool`? Why are lsn_end and layer_file_size optionals?

Reply:
Layer file size is optional for historic reasons (I'll look into removing it). This not being a bool is my fault.

lsn_end is None for image layers.

Reply:
[2f1c44b](https://github.com/neondatabase/neon/pull/7813/commits/2f1c44b8f2789edadf79cb9c41334f65c7baa943)

[04ed6ec](https://github.com/neondatabase/neon/pull/7813/commits/04ed6ec29e91244da137594f290b1e6c5eea6d55)